### PR TITLE
commenter 0.0.1

### PR DESCRIPTION
**Pull request by chrisfuller**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/commenter/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+title: Commenter
```
Comment (koral--) [inline steps/commenter/0.0.1/step.yml]: I would call it `Add GitHub comment` or something like that. `Commenter` is too broad.

Comment (godrei) [reply steps/commenter/0.0.1/step.yml]: I agree with @koral-- , we are working on an update for the step development guideline, one of the updates will cover step title requirements. Pls use this layout for title: `<verb/action> <service/tool>`, like `Add GitHub comment`

File: steps/commenter/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  opts:
+    description: |
+      The owner.
+    is_expand: true
+    is_required: true
+    title: --owner
+- commenter_repo: ${BITRISEIO_GIT_REPOSITORY_SLUG}
+  opts:
+    description: |
+      The repository.
```
Comment (godrei) [inline steps/commenter/0.0.1/step.yml]: pls explain it in more details, the repository usually refers to an url.

File: steps/commenter/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+is_always_run: false
+is_skippable: false
+inputs:
+- commenter_owner: ${BITRISEIO_GIT_REPOSITORY_OWNER}
+  opts:
+    description: |
+      The owner.
+    is_expand: true
+    is_required: true
+    title: --owner
```
Comment (godrei) [inline steps/commenter/0.0.1/step.yml]: pls refer this comment: https://github.com/bitrise-io/bitrise-steplib/pull/2124#discussion_r323202409

File: steps/commenter/0.0.1/step.yml (hunk 1 above)
Comment (chrisfuller) [reply steps/commenter/0.0.1/step.yml]: Fixed in `v0.0.2` 👍

File: steps/commenter/0.0.1/step.yml (hunk 2 above)
Comment (chrisfuller) [reply steps/commenter/0.0.1/step.yml]: Fixed in `v0.0.2` 👍

File: steps/commenter/0.0.1/step.yml (hunk 3 above)
Comment (chrisfuller) [reply steps/commenter/0.0.1/step.yml]: Fixed in `v0.0.2` 👍

**Comment by godrei** [comment]
@chrisfuller please refer this message: https://github.com/bitrise-io/bitrise-steplib/pull/2113#issuecomment-532139355

**Comment by godrei** [comment]
Closing as we did not receive a reply - @chrisfuller please let us know if we can help with anything!

**Pull request closed without merging**